.rspec

```diff
@@ -0,0 +1,2 @@
+--color
+--require spec_helper
```

Comment:
So far, all specs require `rails_helper`, which requires `spec_helper`, so that should be all well and good. Added `.rspec` to `.gitignore`.